customer-book-service.html

```diff
@@ -20,83 +20,94 @@
     <div id="header-placeholder"></div>
 
 
-<main>
-    <section>
-        <h1>Book a service</h1>
-        <div class="service-box">
-        <h3>First Service - Tutoring</h3>
-            <p>Our first service is tailored for kids from 6 to 18 years old. It supports students who need extra help with their studies. We offer private tutoring services. Prices vary by age:</p>
-            <ul>
-                <li>Ages 6 to 10: $20/hour</li>
-                <li>Ages 11 to 16: $30/hour</li>
-                <li>Ages 17 to 18: $40/hour</li>
-            </ul>
-            </p>
-            <button type="button" onclick="bookService()">BOOK SERVICE</button>
-            <hr>
+    <main>
+        <section>
+            <h1>Book a service</h1>
+            <div class="service-box">
+                <h3>First Service - Tutoring</h3>
+                <p>Our first service is tailored for kids from 6 to 18 years old. It supports students who need extra
+                    help with their studies. We offer private tutoring services. Prices vary by age:</p>
+                <ul>
+                    <li>Ages 6 to 10: $20/hour</li>
+                    <li>Ages 11 to 16: $30/hour</li>
+                    <li>Ages 17 to 18: $40/hour</li>
+                </ul>
+                </p>
+                <button type="button" onclick="bookService()">BOOK SERVICE</button>
```

Comment:
the writing is cut off, does it display correctly on website?  like there is a big gap between the paragraphs